BFF.py

```diff
@@ -267,6 +269,22 @@ def get_narrative_users(WorkspaceIdentityList, narrative_list, auth_token):
         narrative['users'] = response_json['result'][0]['perms'][index]
 
 
+@app.route('/fetchUserProfiles', methods=['POST'])
+def get_userProfiles():
+    body = request.get_json()
+    userIDs = body.get("users", [])
+    userProfile_payload = {
+        'id': 0,
```

Comment:
This may be the way the codebase works, but the id should be the id in the request. A strict jsonrpc client may consider this an error.